### PR TITLE
wisconsin http get pull from json

### PR DESCRIPTION
**Pull request by jchen038**

### CODE REVIEW

**Comment by huuep** [comment]
Where did you get the source link? is it from the WI health dept website?

**Comment by jchen038** [comment]
yep they have a restful api page you can play with.

**Comment by huuep** [comment]
can you use that as source? the arcgis source you added is unclear on whether it was from wi.gov

File: states.csv [hunk 1]
```diff
@@ -47,6 +47,6 @@ UT	https://coronavirus-dashboard.utah.gov
 VA	http://www.vdh.virginia.gov/surveillance-and-investigation/novel-coronavirus/
 VT	https://www.healthvermont.gov/response/infectious-disease/2019-novel-coronavirus
 WA	https://www.doh.wa.gov/emergencies/coronavirus
-WI	https://www.dhs.wisconsin.gov/outbreaks/index.htm
+WI	https://opendata.arcgis.com/datasets/b913e9591eae4912b33dc5b4e88646c5_10.geojson
```
Comment (huuep) [inline states.csv:50]: this source is unknown, ie not clear it is from wisconsin's dhs

Comment (jchen038) [reply states.csv:50]: if you go to https://www.dhs.wisconsin.gov/outbreaks/index.htm and click down state data, it leads to this url

Comment (huuep) [reply states.csv:50]: state data took me to: https://data.dhsgis.wi.gov/datasets/covid-19-historical-data-table/data?where=GEO%20%3D%20%27State%27

**Comment by huuep** [comment]
was fixed

**Pull request closed without merging**